A52-xds-custom-lb-policies.md

```diff
@@ -34,7 +34,7 @@ Regardless of the field that load balancing policy configuration is picked up fr
 
 The load balancer config portion of the CDS update struct will change to take the following form:
 
-```json
+```c
```

Comment:
If this is supposed to be C, then I would format it like this:

```
struct XdsClusterResource {
  // ...

  // The gRPC LB policy config to be used as the child of the xds_cluster_impl LB policy.
  Json::Array lb_policy_config;

  // ...
}
```

Reply:
Ok I just switched to this and mentioned that this is what it would look like in C++/